src/main/java/oshi/software/os/OperatingSystem.java

```diff
@@ -48,4 +48,18 @@
      * @return Version.
      */
     OperatingSystemVersion getVersion();
+
+    /**
+     * Open file descriptors.

```

Comment:
Need a much more verbose javadoc at the higher level.  Explain what these are and specifically note that they are not implemented on Windows (because of the way Windows treats file handles.)